ranger-zookeeper/src/main/java/com/flipkart/ranger/zookeeper/util/PathBuilder.java

```diff
@@ -1,12 +1,12 @@
-/**
+/*
  * Copyright 2015 Flipkart Internet Pvt. Ltd.
- *
+ * <p>
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- *
+ * <p>
  * http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p>
```

Comment:
No way to comment belo, replace the `@NoArgs..` annotation and put in `@utilityClass`

Reply:
Part of the lombok cleanup